boards/xtensa/qemu_xtensa/qemu_xtensa_mmu.yaml

```diff
@@ -1,5 +1,5 @@
-identifier: qemu_xtensa_dc233c
```

Comment:
need to add zephyr to the toolchain part...

Reply:
Done.